.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URI=http://localhost:8000/api/
```

Comment:
have  you pulled latest  changes from the `itez-platform`
`git submodule update --remote`
??

this has to read `ITEZ_API_URI`

Reply:
- [x] separated the tickets